app/js/voice.js

```
@@ -35,8 +36,14 @@
                 const hasK = t === 'k' || t.startsWith('k ');
 
                 if ((hasKelion || hasKira || hasK) && event.results[i].isFinal) {
-                    if (hasKira) { window.KAvatar.loadAvatar('kira'); document.querySelectorAll('.avatar-pill').forEach(b => b.classList.toggle('active', b.dataset.avatar === 'kira')); }
-                    else { window.KAvatar.loadAvatar('kelion'); document.querySelectorAll('.avatar-pill').forEach(b => b.classList.toggle('active', b.dataset.avatar === 'kelion')); }
+                    const targetAvatar = hasKira ? 'kira' : 'kelion';
+                    const currentAvatar = window.KAvatar.getCurrentAvatar();
+                    if (targetAvatar !== currentAvatar) {
+                        window.KAvatar.loadAvatar(targetAvatar);
+                        document.querySelectorAll('.avatar-pill').forEach(b => b.classList.toggle('active', b.dataset.avatar === targetAvatar));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)